### DISCUSSION TITLE
Should I wait for v4 to build my extension or just go with v3?

### DISCUSSION THREAD

**Discussion by grandslammer** [crxjs/chrome-extension-tools#192] [Q&A] [answered]
I'm currently building a large, complex extension using Svelte. I am currently building it as a regular Svelte website and plan to migrate it into becoming an extension soon using rollup-plugin-chrome-extension.

I have been following rollup-plugin-chrome-extension for a while and am aware that v4 is currently in beta.

What would you recommend me to do - use the current v3 version, or wait for v4 to release?

If I was to go with v3 for now, is it simple to update to v4 when it hits 1.0?

What's the ETA on v4?

Thank you in advance.

**Comment by jacksteamdev** [comment, marked as answer]
A pretty stable version of v4 should hit this week. 🤞

I will release a migration guide for v3 users. It won't be complex.
